tests/outputs/TestPar.cpp

```diff
@@ -244,7 +244,7 @@ TEST(TestPar, writeStaticVarCompensator) {
   outputPath.append(filename);
   dfl::inputs::Configuration::ActivePowerCompensation activePowerCompensation(dfl::inputs::Configuration::ActivePowerCompensation::P);
   dfl::outputs::Par parWriter(dfl::outputs::Par::ParDefinition(basename, dirname, outputPath.generic_string(), {}, {}, activePowerCompensation, {},
-                                                               manager, {}, {}, {}, svarcs));
+                                                               manager, {}, {}, {}, svarcs, {}));
```

Comment:
Maybe adding a test with no load to test that the connection is not dumped? (same goes for dyd)

Reply:
I'd rather add a test on loads as missing, for the other tests loads are already missing